app/src/controllers/pagination/withPagination.jsx

```diff
@@ -63,12 +67,14 @@ export const withPagination = ({
     changeSizeHandler = (size) => this.changePaginationOptions({ size, page: 1 });
 
     changePaginationOptions = (options) => {
-      const { page, size } = this.props;
+      const { userId, page, size, ...rest } = this.props;
```

Comment:
`const { userId, page, size, namespace, ...rest } = this.props;`

Reply:
Fixed.

Reply:
Fixed.